src/sdg/api/views/organisaties.py

```diff
@@ -50,11 +52,21 @@ class LokaleOverheidViewSet(viewsets.ReadOnlyModelViewSet):
     retrieve=extend_schema(
         description="Locatie waar de bijbehorende organisatie 1 of meer producten aanbiedt."
     ),
+    create=extend_schema(description="Maak een nieuwe locatie voor een organisatie."),
+    update=extend_schema(description="Update de locatie van een organistatie."),
+    destroy=extend_schema(description="Verweider de locatie van een organisatie"),
 )
-class LocatieViewSet(viewsets.ReadOnlyModelViewSet):
+class LocatieViewSet(viewsets.ModelViewSet):
     """Viewset for a location, retrieved by uuid"""
 
     lookup_field = "uuid"
     queryset = Locatie.objects.select_related("lokale_overheid")
     filterset_class = LocatieFilterSet
     serializer_class = LocatieSerializer
+    permission_classes = [LocationPermission]
+
+    def get_organisatie(self, request, view, obj=None):
+        if request.method == "POST":
+            return view.request.data.get("organisatie")
+
+        return obj.lokale_overheid.organisatie
```

Comment:
This function should always return the same type of value. So please refactor to something like:

```suggestion
    def get_organisatie(self, request, view, obj=None):
        if request.method == "POST":
            return Organisatie.objects.get(id=view.request.data.get("organisatie"))

        return obj.lokale_overheid.organisatie
```